report/report_handler.go

```
@@ -0,0 +1,79 @@
+package report
+
+import (
+	"github.com/hanochg/piperika/http"
+	"github.com/hanochg/piperika/http/requests"
+	"strconv"
+	"strings"
+)
+
+const adHocReleaseBranchName = "AD_HOC_RELEASE_BRANCH_NAME"
+
+func getPipelineReport(httpClient http.PipelineHttpClient, serviceName string, pipeSuffix string, branch string, pipeline *PipelineResult) {
+	if branch == "" {
+		pipeline.Status = "NotCreated"
+		return
+	}
+
+	pipeline.Name = serviceName + pipeSuffix
+	pipeline.Branch = branch
+
+	pipeResp, _ := requests.GetPipelines(httpClient, requests.GetPipelinesOptions{
```

Do not ignore errors

---

Same for other http requests